proto/dymensionxyz/dymension/rollapp/rollapp.proto

```diff
@@ -17,31 +19,51 @@ message RollappGenesisState {
   bool transfers_enabled = 2;
 }
 
-// Rollapp defines a rollapp object. First the RollApp is created and then
+// Rollapp defines a rollapp object. First, the RollApp is created and then
 // sequencers can be created and attached. The RollApp is identified by rollappId
 message Rollapp {
   // The unique identifier of the rollapp chain.
-  // The rollappId follows the same standard as cosmos chain_id.
-  string rollappId = 1;
+  // The rollapp_id follows the same standard as cosmos chain_id.
+  string rollapp_id = 1;
   // creator is the bech32-encoded address of the rollapp creator.
   string creator = 2;
-  // version is the software and configuration version.
-  // starts from 1 and increases by one on every MsgUpdateState
-  uint64 version = 3;
-  // maxSequencers is the maximum number of sequencers.
-  uint64 maxSequencers = 4;
-  // permissionedAddresses is a bech32-encoded address list of the sequencers that are allowed to serve this rollappId.
-  // In the case of an empty list, the rollapp is considered permissionless.
-  repeated string permissionedAddresses = 5;
-  reserved 6;
+  // initial_sequencer_address is a bech32-encoded address of the
+  // sequencer that is allowed to initially serve this rollappId.
+  string initial_sequencer_address = 3;
```

Comment:
can't u just rename `permissionedAddresses` and enforce it on genesis sequencer creation only?

Reply:
hmm... `persmissionedAddresses` is a list of strings, how will that unmarshal onto the new string field?